nautobot/core/tests/integration/test_view_authentication.py

```diff
@@ -0,0 +1,67 @@
+from django.test import tag
+
+from nautobot.core.testing import TestCase
+from nautobot.core.utils.lookup import get_url_for_url_pattern, get_url_patterns
+
+
+@tag("integration")
+class AuthenticationEnforcedTestCase(TestCase):
+    r"""
+    Test that all\* registered views require authentication to access.
+
+    \* with a very small number of known exceptions such as login and logout views.
```

Comment:
Why the raw docstring with the escaped asterisks?

Reply:
Thinking that at some point we'll treat docstrings as Markdown (as we already do for stuff in the apps namespace. Didn't want the asterisks to map to bold.